lawlibrary/templates/peachjam/legislation_detail.html

```diff
@@ -3,14 +3,25 @@
 {% block breadcrumbs %}
   <nav aria-label="breadcrumb">
     {% if document.locality %}
-      <ol class="breadcrumb">
-        <li class="breadcrumb-item">
-          <a href="{% url 'locality_legislation' %}">{% trans 'Provincial Legislation' %}</a>
-        </li>
-        <li class="breadcrumb-item">
-          <a href="{% url 'locality_legislation_list' document.locality.code %}">{{ document.locality.name }}</a>
-        </li>
-      </ol>
+      {% if document.locality.code in PROVINCIAL_CODES %}
```

Comment:
Only the first part of the breadcrumb changes, the rest is the same. So put the if block only around the part that changes, otherwise you're duplicating code.